usage-based-subscriptions/server/go/report_usage.go

```diff
@@ -0,0 +1,32 @@
+package main
```

Comment:
I'm not sure what the `imports` should look like for this. Lines 10+ were copy and pasted.

Reply:
I think this looks okay to me!